### ISSUE TITLE
Cant download prisma engines

### ISSUE THREAD

**Issue by Kolhax** [AlphaNecron/Void#46]
Error:
```
kepar@kepar:~/disk2/Void$ npx prisma generate
Prisma schema loaded from prisma/schema.prisma
> Downloading Prisma engines for Node-API for debian-openssl-3.0.x [                    ] 0%Error: Not Found https://binaries.prisma.sh/all_commits/73e60b76d394f8d37d8ebd1f8918c79029f0db86/debian-openssl-3.0.x/libquery_engine.so.node.gz
kepar@kepar:~/disk2/Void$
```

**Comment by Kolhax** [comment]
```
kepar@kepar:~/disk2/Void$ node -v
v18.9.0
kepar@kepar:~/disk2/Void$ npx prisma --version
prisma                  : 4.3.1
@prisma/client          : 3.10.0
Current platform        : debian-openssl-3.0.x
Query Engine (Node-API) : libquery-engine c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/libquery_engine-debian-openssl-3.0.x.so.node)
Migration Engine        : migration-engine-cli c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/migration-engine-debian-openssl-3.0.x)
Introspection Engine    : introspection-core c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/introspection-engine-debian-openssl-3.0.x)
Format Binary           : prisma-fmt c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/prisma-fmt-debian-openssl-3.0.x)
Format Wasm             : @prisma/prisma-fmt-wasm 4.3.0-32.c875e43600dfe042452e0b868f7a48b817b9640b
Default Engines Hash    : c875e43600dfe042452e0b868f7a48b817b9640b
Studio                  : 0.473.0
```

**Comment by Kolhax** [comment]
FIXED BY DOWNGRADING OPENSSL TO 1.1.x
